src/lab-tiles/laboratory-summary-tiles.scss

```diff
@@ -5,13 +5,11 @@
 .cardContainer {
   background-color: $ui-02 ;
   display: flex;
-  padding: 0 spacing.$spacing-05 spacing.$spacing-10 spacing.$spacing-03;
+  padding: 0 spacing.$spacing-01 spacing.$spacing-09 spacing.$spacing-03;
 }
 
 .cardContainer > div:nth-child(1),
 .cardContainer > div:nth-child(2),
-.cardContainer > div:nth-child(3),
-.cardContainer > div:nth-child(4),
-.cardContainer > div:nth-child(5) {
-  width: 20%;
+.cardContainer > div:nth-child(3) {
+  width: 33%;
 }
```

Comment:
Rather than doing this, this is a perfect case for using a CSS Flexbox. This hard codes "there are exactly three metric tiles". See [this tutorial](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) for a decent overview.